homeassistant/components/cast/media_player.py

```diff
@@ -526,6 +492,34 @@ async def async_browse_media(self, media_content_type=None, media_content_id=Non
             children=children,
         )
 
+    async def async_browse_media(self, media_content_type=None, media_content_id=None):
+        """Implement the websocket media browsing helper."""
+        content_filter = None
+
+        if self._chromecast.cast_type == pychromecast.const.CAST_TYPE_AUDIO:
+
+            def audio_content_filter(item):
+                """Filter non audio content."""
+                return item.media_content_type.startswith("audio/")
+
+            content_filter = audio_content_filter
+
+        if media_content_id is None:
+            return await self._async_root_payload(content_filter)
+
+        for _, platform in self.hass.data[CAST_DOMAIN].items():
+            if not hasattr(platform, "async_browse_media"):
+                continue
+            browse_media = await platform.async_browse_media(
```

Comment:
We should indicate here that it's audio only or not.

Reply:
Fixed in 10ffd80, we now pass a filtering function